bin/resources/GameIndex.yaml

```diff
@@ -4772,6 +4772,27 @@ SCPM-85101:
 SCPM-85301:
   name: "Gran Turismo Concept - Copen Special Edition [Demo]"
   region: "NTSC-J"
+SCPN-60101:
+  name: "PlayStation BB Navigator - Version 0.10 [Prerelease] [Disc 1]"
+  region: "NTSC-J"
+SCPN-60103:
+  name: "PlayStation BB Navigator - Version 0.10 [Prerelease] [Disc 2]"
+  region: "NTSC-J"
+SCPN-60111:
+  name: "PlayStation BB Navigator - Version 0.10 [Prerelease] [Disc 2]"
```

Comment:
Where's disc 1 for this one?

Reply:
Disc 1 (SCPN-60101) is common for both releases

Reply:
These Disc 2s are not bootable, They only have a config file. Should I delete them?

Reply:
Nah it's fine, it makes the DB more complete.